Tests/SwiftLintFrameworkTests/QuickDiscouragedCallRuleTests.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  QuickDiscouragedCallRuleTests.swift
+//  SwiftLint
+//
+//  Created by Omer Murat Aydin on 28.12.2024.
+//
+
+@testable import SwiftLintBuiltInRules
+import XCTest
+@testable import SwiftLintFramework
+
+class QuickDiscouragedCallRuleTests: XCTestCase {
```

Comment:
This new test class is not required. Please use `QuickDiscouragedCallRuleExamples` instead to add your test cases.